.github/workflows/build.yml

```diff
@@ -207,6 +207,9 @@ jobs:
           node-version: ${{ needs.setup-build.outputs.node-version }}
           os: ${{ runner.os }}
 
+      - name: Generate protos
+        run: pnpm create-protos
+
```

Comment:
revert those changes and do it in generate script, see package.json for other generate scripts

Reply:
That was my first approach. I tried doing it in generate but that creates a dependency issue and requires modifying a number of test commands and likely the github workflows to run generate where they currently do not. One example is linting which does not have a dependency on `setup` currently. This causes the files to not be present when there are references to them.

<img width="547" alt="image" src="https://github.com/user-attachments/assets/baf36835-009d-4f76-8289-de3ee5aa96e6">

Introducing a generic protos creation step was the least intrusive solution I could get to.

If you can assist in getting to a working solution with generate, it would be much appreciated.